src/react-dnd-ax/drag-n-drop-item.js

```diff
@@ -29,6 +35,8 @@ const DragNDropItem = (WrappedComponent) => {
     componentDidMount() {
       const {actions} = this.props
 
+      window.addEventListener("resize", this.resetPreviewWidth)
```

Comment:
Depending on your requirement, can we use `throttle` or `debounce` to `this.resetPreviewWidth` to avoid high frequent calls.